openquake.cfg

```diff
@@ -47,6 +47,11 @@ host = localhost
 port = 1999
 authkey = changeme
 
+[directory]
+# the base directory containing the <user>/oqdata directories
```

Comment:
I would add that the default is `<user_home>/oqdata`, otherwise it is `shared_dir/<user>/oqdata`